internal/transactionsubmission/manager_test.go

```diff
@@ -448,6 +461,9 @@ func Test_Manager_ProcessTransactions(t *testing.T) {
 			chTxBundleModel, err := store.NewChannelTransactionBundleModel(dbConnectionPool)
 			require.NoError(t, err)
 
+			mMonitorClient := &monitorMocks.MockMonitorClient{}
+			mMonitorClient.On("MonitorCounters", mock.Anything, mock.Anything).Return(nil)
+
```

Comment:
💡  For mocks, we usually signal how many times we expect them to be called, and we also call `AssertExpectations` at the end of the methods. Like in:

https://github.com/stellar/stellar-disbursement-platform-backend/blob/8d98f3d244a038088676dba69c5238b439640d4b/internal/transactionsubmission/horizon_test.go#L142-L143

and:
https://github.com/stellar/stellar-disbursement-platform-backend/blob/8d98f3d244a038088676dba69c5238b439640d4b/internal/transactionsubmission/horizon_test.go#L190